sys/include/net/ieee802154/radio.h

```diff
@@ -190,6 +194,26 @@ typedef enum {
      */
     IEEE802154_RADIO_INDICATION_RX_START,
 
+    /**
+     * @brief the transceiver received a frame with an invalid crc
+     *        and lies in the internal framebuffer.
+     *
+     * This indication should be generated only if the CRC is invalid and
+     * the frame passes the address matching filter (this includes ACK
+     * and Beacon frames). The latter only applies if the radio is not
+     * in promiscuous mode.
+     *
+     * The transceiver is in @ref IEEE802154_TRX_STATE_RX_ON state when
+     * this function is called, but with framebuffer protection (either
+     * dynamic framebuffer protection or disabled RX). Thus, the frame
+     * won't be overwritten before calling @ref ieee802154_radio_indication_crc_error
+     * function. However, @ref ieee802154_radio_indication_crc_error
```

Comment:
We removed the "indication" procedure in https://github.com/RIOT-OS/RIOT/pull/15228. I would suggest to do the same here (e.g state that `ieee802154_radio_request_set_trx_state` should be called in case this is received)

Reply:
Ah sorry about that, will update.

Reply:
What about:

```
     * The transceiver is in @ref IEEE802154_TRX_STATE_TRXOFF state when
     * this indication is generated. Thus, if the radio should listen
     * for new frames the radio must be switched to @ref IEEE802154_TRX_STATE_RX_ON
     * by calling @ref ieee802154_radio_request_set_trx_state()
```